python/sparknlp/internal/py4j_utils.py

```diff
@@ -0,0 +1,32 @@
+# Coloring the print
+FAIL = '\033[91m'
+ENDC = '\033[0m'
```

Comment:
The name of these variables is a little confusing. To make it more readable, we could rename those e.g.
```
FAIL to FAIL_COLOR
ENDC to END_COLOR
```